commons/src/main/java/org/infinispan/commons/configuration/attributes/AttributeDefinition.java

```diff
@@ -69,6 +74,10 @@ public boolean isAutoPersist() {
       return autoPersist;
    }
 
+   public boolean isLocal() {
```

Comment:
the IDE says that the result is always negated. maybe, instead of creating a `local` flag, a `global` flag is better :).